### ISSUE TITLE
[Bug] vela top show abnormal  on Kylin OS

### ISSUE THREAD

**Issue by StevenLeiZhang** [kubevela/kubevela#6076]
**Describe the bug**

`vela top`  shows abnormal on Kylin OS.
Vela version: 1.8.2
OS version:  Kylin10.x86_64

![image](https://github.com/kubevela/kubevela/assets/14890978/3e4d68e4-e70d-44cb-ae89-624a4a5f6615)

**To Reproduce**

**Expected behavior**

**Screenshots**

**KubeVela Version**

**Cluster information**

**Additional context**

**Comment by Hanmengnan** [comment]
This may seem like a character set related issue, but I'm not sure if it's related to your terminal or operate system? Have you tried connecting to this device using other terminals and then using vela top?

**Comment by Hanmengnan** [comment]
BTW, could you use `locale` command to check the settings of character encoding? You can run the command `export LANG="en_US.UTF-8"`. Set character encoding to UTF-8.

**Comment by StevenLeiZhang** [comment]
> BTW, could you use `locale` command to check the settings of character encoding? You can run the command export `LC_CTYPE="en_US.UTF-8"`. Set character encoding to UTF-8.

![image](https://github.com/kubevela/kubevela/assets/14890978/3017feca-99c7-45d5-9f96-b2c4ca78c338)

**Comment by StevenLeiZhang** [comment]
> This may seem like a character set related issue, but I'm not sure if it's related to your terminal or operate system? Have you tried connecting to this device using other terminals and then using vela top?

As  you know, Kylin V10 is a kind of Linux, so I just use its Terminal not 3rd party  terminal tool

**Comment by Hanmengnan** [comment]
> > This may seem like a character set related issue, but I'm not sure if it's related to your terminal or operate system? Have you tried connecting to this device using other terminals and then using vela top?
> 
> As you know, Kylin V10 is a kind of Linux, so I just use its Terminal not 3rd party terminal tool

Yes, I understand. The reason I have this question is because I have encountered another display issue before, which was when the user used Vela top using the finalshell. The display error was caused by a problem with this terminal. So I need to confirm first whether you are using the system terminal or connecting to the system through SSH through other terminals.

**Comment by Hanmengnan** [comment]
Since I don't have a KylinOS machine at hand, it's not very convenient to locate the problem.

**Comment by StevenLeiZhang** [comment]
> system terminal

I use system terminal

**Comment by StevenLeiZhang** [comment]
k9s works well on this OS

**Comment by Hanmengnan** [comment]
> k9s works well on this OS

ok, I get it.

**Comment by briankane** [comment]
This issue has been inactive for over 2 years and is scheduled for closure as part of our backlog maintenance.

  **If this issue is still relevant:**
  - Please comment below to keep it open
  - Provide updated context if the original description is outdated

  **Timeline:** If there's no activity by **Friday, October 24th**, this issue will be automatically closed.

  **Note:** Issues can always be reopened if they become relevant again. We encourage the community to reopen and reprioritize any incorrectly 
  closed issues.

**Comment by briankane** [comment]
This issue has been closed following the stale issue notice. 

If this issue is still relevant and should remain open, please:
1. Reopen the issue
2. Remove the 'stale' label
3. Provide an update on the current status or importance

Thank you for your understanding and contribution to KubeVela!
